src/actions/actions/UserData.js

```diff
@@ -82,12 +86,48 @@ export const loginUser = (account, password) => {
             _keys[activeCoins[i].id] = makeKeyPair(seed, activeCoins[i].id)
           }
         }
-        let paymentMethods = {}
-        if (account.paymentMethods) {
-          paymentMethods = JSON.parse(decryptGeneral(seed, account.paymentMethods))
+
+        let hashedSeed = null;
+        let wyreService = WyreService.build();
+        const lengthOfPrefix = 8;
+
+        let paymentMethods = {
+          ...account.paymentMethods
         }
-        let _activeAccount = {id: account.id, wifKey: seed, keys: _keys, paymentMethods}
-        resolve(signIntoAccount(_activeAccount))
+
+        //get data from for account.id async storage
+        getData(account.id).then(dataFromStorage => {
```

Comment:
I have a feeling we don't need this part.

Can we just do this
hashedSeed = sha256(seed).toString('hex');
wyreService.submitAuthToken(hashedSeed)
basedon the response
             paymentMethods.wyre = {
                id: (submitAuthTokenResponse.authenticatedAs ?
                  submitAuthTokenResponse.authenticatedAs.substring(lengthOfPrefix, lengthOfKey) : null),
                key: hashedSeed
              }

Another note, you can use js slice to remove the prefix.